lib/src/components/NxTab/NxTab.scss

```diff
@@ -6,16 +6,49 @@
  */
  @import '../../scss-shared/nx-colors';
  @import '../../scss-shared/nx-variables';
+ @import '../../scss-shared/nx-text-helpers';
 
 .nx-tab {
-  border-bottom: 4px solid transparent;
+  border-bottom: 2px solid transparent;
   cursor: pointer;
   display: inline-block;
   list-style: none;
-  padding: $nx-spacing-xs $nx-spacing-l;
+  margin: 0 $nx-spacing-md -1px 0;
+  // subtract border thickness for consistent spacing
+  padding: $nx-spacing-md $nx-spacing-l ($nx-spacing-md - 2px) $nx-spacing-l;
+  text-align: center;
+  min-width: 76px;
```

Comment:
Needs `box-sizing: border-box` in order to do what was desired.

Reply:
> Needs `box-sizing: border-box` in order to do what was desired.

https://github.com/sonatype/sonatype-react-shared-components/pull/153/commits/2d9f2cb8b2c0a7a0799718bbc70082fc0160d4e4